job-shop-scheduling/demo.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+from dwave.system.composites import EmbeddingComposite
+from dwave.system.samplers import DWaveSampler
+
+from job_shop_scheduler import get_jss_bqm
+
+# Construct a BQM for jobs 'a', 'b' and 'c'
+jobs = {"a": [("mixer", 2), ("oven", 1)],
+        "b": [("mixer", 1)],
+        "c": [("oven", 2)]}
+max_time = 4	  # Upperbound on how long the schedule can be
+bqm = get_jss_bqm(jobs, max_time)
+
+# Submit BQM
+# Note: may need to tweak the chain strength and the number of reads
+sampler = EmbeddingComposite(DWaveSampler(solver={'qpu':True}))
+sampleset = sampler.sample(bqm, chain_strength=2, num_reads=1000)
+
+# Grab solution
+solution = sampleset.first.sample
+
+# Visualize solution
+# Note0: each node in our BQM is labelled as "<job>_<task_index>,<time>".
+#  For example, the node "a_1,2" refers to job 'a', its 1st task (where we are
+#  using zero-indexing, so task '("oven", 1)'), starting at time 2.
+#
+#  Hence, we are grabbing the nodes selected by our solver (i.e. nodes flagged
+#  with 1s) that will make a good schedule. (see 'selected_nodes')
+#
+# Note1: we are making the solution simpler to interpret by restructuring it
```

Comment:
How about swapping the note order so users first learn your intent and then see the details of how to accomplish that?